### ISSUE TITLE
GenL10nException: The 'l10n.yaml' file does not exist

### ISSUE THREAD

**Issue by ska2519** [localizely/flutter-ota-sample-app#5]
when launch
`flutter packages pub run build_runner build --delete-conflicting-outputs`

```
➜ flutter packages pub run build_runner build --delete-conflicting-outputs
[INFO] Generating build script...
[INFO] Generating build script completed, took 209ms

[INFO] Initializing inputs
[INFO] Reading cached asset graph...
[INFO] Reading cached asset graph completed, took 432ms

[INFO] Checking for updates since last build...
[INFO] Checking for updates since last build completed, took 680ms

[INFO] Running build...
[INFO] Running build completed, took 29ms

[INFO] Caching finalized dependency graph...
[INFO] Caching finalized dependency graph completed, took 190ms

[SEVERE] localizely_sdk:localizely_builder on $package$ (cached):

GenL10nException: The 'l10n.yaml' file does not exist.
[SEVERE] Failed after 227ms
pub finished with exit code 1
```

i got this Exception.
I'm setting OTA service with out  'l10n.yaml'.

how to fix it?

**Comment by lzoran** [comment]
Hi @ska2519,

Thanks for reporting this!

The `localizely_sdk 2.4.0` added support for the In-Context Editing feature and thus added builder config for generating localization code for the `gen_l10n` tool.

Not sure how you use `build_runner` in your project, but maybe these two workarounds can help you:

- Downgrade `localizely_sdk` to `2.3.0` in case you only need Flutter Over-the-Air (support for Flutter In-Context Editing is added in `2.4.0`)
- Exclude `localizely_sdk:localizely_builder` from the build process or invoke your build script independently of other packages (if possible)

**Comment by ska2519** [comment]
Hi @lzoran 
Thanks for kindness reply :)
i will change by `2.3.0` version.

**Comment by lzoran** [comment]
In case someone encounters this issue, it is fixed with the `localizely_sdk ^2.4.1`.
